Packs/PICUS/Integrations/PICUS/PICUS.py

```diff
@@ -0,0 +1,901 @@
+    page = int(demisto.args().get('page'))
+    size = int(demisto.args().get('size'))
+    data = {'add_user_details': add_user_details,
+            'page': page,
+            'size': size
+            }
+
+    req = requests.post(url, headers=headers, data=json.dumps(data), verify=VERIFY_SSL)
+    parsed = json.loads(req.content)
+
+    results = parsed['data']['vectors'] if 'data' in parsed else ''
+
+    header = ['name', 'description', 'trusted', 'untrusted', 'is_disabled', 'type']
+    hr = tableToMarkdown('Picus Vector List', results, header, removeNull=True)
+
+    return {
+        'Type': entryTypes['note'],
+        'Contents': results,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': hr
+    }
+
+
+'''Response:
+{
+  "data": {
+    "page_number": 2,
+    "size": 5,
+    "total_count": 548,
+    "vectors": [
+      {
+        "description": "string",
+        "heartbeat_results": [
+          {
+            "is_successful": true,
+            "module": "PTS",
+            "result_time": "2018-08-21T13:00:33.590599407Z",
+            "variant": "HTTP"
+          }
+        ],
+        "is_disabled": true,
+        "name": "Email-Peer-2 - Email-Peer-1",
+        "trusted": "Windows10-Peer",
+        "type": "string",
+        "untrusted": "Network-Peer",
+        "users": [
+          {
+            "role": "admin",
+            "username": "oliver"
+          }
+        ]
+      }
+    ]
+  },
+  "success": true
+}
+'''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    ''' EXECUTION '''
+    #LOG('command is %s' % (demisto.command(), ))
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        LOG('Command being called is {command}'.format(command=demisto.command()))
+        if demisto.command() == 'Picus-GetAccessToken':
+            getAccessToken()
+        elif demisto.command() == 'Picus-Vector-Compare':  # Makes a comparison of the given vector's results
+            token = getAccessToken()
+            demisto.results(vectorCompare(token))
+        elif demisto.command() == 'Picus-Attack-Result-List':  # Returns the list of the attack results\nhave optional parameters for pagination and filtration
+            token = getAccessToken()
+            demisto.results(attackResultList(token))
+        elif demisto.command() == 'Picus-Specific-Threats-Results':  # Returns the list of the attack results of a single threat\nhave optional
+            token = getAccessToken()
+            demisto.results(specificThreatsResults(token))
+        elif demisto.command() == 'Picus-Peer-List':  # Returns the peer list with current statuses
+            token = getAccessToken()
+            demisto.results(peerList(token))
+        elif demisto.command() == 'Picus-EMail-Peer-List':  # Returns the E-Mail peer list with current statuses
```

Comment:
Notice that the YML doesn't state that there is a command like that.
In case you would like to add it to the YML, use the name convention below.

```suggestion
        elif demisto.command() == 'picus-email-peer-list':  # Returns the E-Mail peer list with current statuses
```

Reply:
Resolved.